workflow.md

```diff
@@ -112,9 +112,8 @@ release must be described in the corresponding section.
 
 Each section contains 3 subsections:
```

Comment:
With this we would have `2 subsections`(not `3`) now. However, I would not remove `News`, but rename `News` to `Features`.